met-web/src/components/engagement/form/ActionContext.tsx

```diff
@@ -30,9 +30,9 @@ export const ActionContext = createContext<EngagementContext>({
     handleUpdateEngagementRequest: (_engagement: EngagementFormUpdate): Promise<Engagement> => {
         return Promise.reject();
     },
-    handleCreateEngagementMetadataRequest: (_engagement: EngagementMetadata): Promise<EngagementMetadata> => {
-        return Promise.reject();
-    },
+    // handleCreateEngagementMetadataRequest: (_engagement: EngagementMetadata): Promise<EngagementMetadata> => {
```

Comment:
Usually with commented-out code, we have a TODO comment explaining why this is here or we remove it.

Reply:
I'll try to stub these methods out more completely.